app/views/cases/filters/retention.html.slim

```diff
@@ -0,0 +1,86 @@
+
+- content_for :page_title do
+  = t('page_title.closed_cases_page')
+
+- content_for :heading
+  = t('.heading')
+
+= render partial: 'layouts/header'
+
+= render partial: 'cases/shared/search_bar'
+
+= render partial: 'cases/shared/case_tabs'
+
+section.govuk-tabs__panel
+  - if @query.available_filters(current_user, @current_tab_name)
+    = render partial: 'cases/search_filters/filters'
+
+    - if @query.filter_crumbs.present?
+      = render partial: 'cases/search_filters/filter_crumbs',
+               locals: { query: @query,
+                         clear_params: {} }
+
+  - if @global_nav_manager.current_page.tabs.present?
+    
+    = get_cases_order_option_url(request.fullpath, cookies[:search_result_order])
+    .button-holder
+      = submit_tag "Further review needed", class: 'button-secondary', form: 'retention_schedules_form' 
+      = submit_tag "Retain", class: 'button-secondary', form: 'retention_schedules_form' 
+      = submit_tag "Mark for destruction", class: 'button-secondary', form: 'retention_schedules_form' 
+      = submit_tag "Destroy cases", class: 'button-secondary', form: 'retention_schedules_form' 
+
+    .grid-row
+      .column-full
+        nav.section-tabs
+          ul
+            - @global_nav_manager.current_page.tabs.each do |tab|
+              li class="tab #{active_link_class(tab.fullpath)}"
+                = link_to t("nav.tabs.#{tab.name}",
+                           count: tab.count),
+                          tab.fullpath_with_query
+    - if @cases.present?
+        .grid-row
+          .column-full.table-container.container.cases-table-container tabindex="0"
+            table.report.table-font-xsmall
+              colgroup
+                col
+                col
+                col
+                col
+                col
+                col
+                col
+              thead
+                th scope='col'
```

Comment:
I think we don't need scope here since it's a simple table.

Reply:
Great thank you i'll remove 👍